MSAL/src/requests/MSALInteractiveRequest.m

```diff
@@ -176,77 +104,79 @@ - (void)acquireToken:(MSALCompletionBlock)completionBlock
     }];
 }
 
+
 - (void)acquireTokenImpl:(MSALCompletionBlock)completionBlock
 {
-    NSURL *authorizationUrl = [self authorizationUrl];
-    
-    MSID_LOG_INFO(_parameters, @"Launching Web UI");
-    MSID_LOG_INFO_PII(_parameters, @"Launching Web UI with URL: %@", authorizationUrl);
-    s_currentRequest = self;
+    MSIDWebviewConfiguration *config = [[MSIDWebviewConfiguration alloc] initWithAuthorizationEndpoint:_authority.authorizationEndpoint
+                                                                                           redirectUri:_parameters.redirectUri
+                                                                                              clientId:_parameters.clientId resource:nil
+                                                                                                scopes:_parameters.scopes
+                                                                                         correlationId:_parameters.correlationId
+                                                                                            enablePkce:YES];
+    config.promptBehavior = MSALParameterStringForBehavior(_uiBehavior);
+    config.loginHint = _parameters.loginHint;
```

Comment:
Need to set `config.extraQueryParameters` as well? :)

Reply:
yes thanks